interfaces/Portalicious/src/app/pages/project-payments/components/payment-summary-card/payment-summary-card.component.html

```diff
@@ -0,0 +1,80 @@
+<p-card
+  styleClass="[&_.p-card-content]:flex-end group relative h-full border border-transparent *:h-full focus-within:border-purple-500 hover:border-purple-500 [&_.p-card-title]:flex-grow"
+>
+  <ng-template pTemplate="title">
+    <a
+      [routerLink]="paymentLink(projectId(), paymentId())"
+      class="after:absolute after:inset-0 after:block group-focus-within:text-purple group-hover:text-purple"
+    >
+      @if (!paymentId()) {
+        <app-skeleton-inline />
+      } @else {
+        <span
+          class="line-clamp-2 text-ellipsis"
+          [title]="
+            'Payment ' + (paymentDate() | date: 'dd-MM-yyyy, hh:mm')
+              | translatableString
+          "
+        >
+          {{
+            'Payment ' + (paymentDate() | date: 'dd-MM-yyyy, hh:mm')
+              | translatableString
+          }}
+        </span>
+      }
+    </a>
+  </ng-template>
+  <ng-template pTemplate="content">
+    <dl class="flex flex-wrap divide-y border-b border-grey-300">
+      <div class="basis-1/2 border-t border-grey-300 px-1 py-2">
+        <app-metric-container
+          [pending]="metrics.isPending()"
+          [value]="includedRegistrations()"
+          label="Included reg."
+          i18n-label
+        ></app-metric-container>
+      </div>
+      <div class="basis-1/2 border-s border-grey-300 px-1 py-2">
+        <app-metric-container
+          [pending]="metrics.isPending()"
+          [value]="totalAmount()"
+          label="Total amount"
+          i18n-label
+        ></app-metric-container>
+      </div>
+      <div class="basis-1/2 border-t border-grey-300 px-1 py-2">
+        <app-metric-container
+          [pending]="metrics.isPending()"
+          [value]="metrics.data()?.success?.count"
+          label="Successful transfers"
+          i18n-label
+        ></app-metric-container>
+      </div>
+      <div class="basis-1/2 border-s border-t border-grey-300 px-1 py-2">
+        <app-metric-container
+          [pending]="metrics.isPending()"
+          [value]="metrics.data()?.failed?.count"
+          [showAlert]="showFailedAlert()"
+          label="Failed transfers"
+          i18n-label
+        ></app-metric-container>
+      </div>
+    </dl>
+  </ng-template>
+  <ng-template pTemplate="footer">
+    <div class="min-h-8">
+      @if (showPaymentInProgressChip()) {
+        @if (metrics.isPending()) {
+          <app-skeleton-inline width="8rem" />
+        } @else {
```

Comment:
With your current logic, I think the skeleton never shows, because `showPaymentInProgressChip()` is only true when `paymentInProgress` has finished loading.

I think this should work instead (and then the `showPaymentInProgressChip` computed property would be obsolete): 

```suggestion
      @if (cardIndex() === 0) {
        @if (paymentInProgress.isPending()) {
          <app-skeleton-inline width="8rem" />
        } @else if (paymentInProgress.data()) {
```